### DISCUSSION TITLE
How to call services on alarm_control_panel

### DISCUSSION THREAD

**Discussion by timvdkerkhof** [fuatakgun/eufy_security#357] [Q&A]
Hi,
I recently bought a Eufy keypad to arm and disarm my alarm. In the Eufy app, there are two different states (Off and Disabled) which translate to the same home assistant state (Disarmed). Also when I disarm the alarm by calling the home assistant service alarm_control_panel.alarm_disarm, the state in the Eufy app changes to Disabled while I would really like to switch to the Off state instead.
In the plugin code, I can see a service being registered for alarm_off, which switches the Eufy state to Off (https://github.com/fuatakgun/eufy_security/blob/master/custom_components/eufy_security/alarm_control_panel.py). 
Is it possible to call this service in home assistant? I tried calling alarm_control_panel.alarm_off, but this returns an error saying that the service cannot be found.

Edit: I found that I need to call eufy_security.alarm_off to set the state to Off. However, this results in the following error messages in the add-on logs:

> InvalidPropertyValueError  Value "6" isn't a valid value for property "guardMode"
> error stack:
> • utils.ts:114 validValue
>     node_modules/eufy-security-client/src/utils.ts:114:19
> • station.ts:484 setGuardMode
>     node_modules/eufy-security-client/src/http/station.ts:484:19
> • message_handler.ts:22 handle
>     src/lib/station/message_handler.ts:22:31
> • server.ts:37 station
>     src/lib/server.ts:37:35
> • server.ts:103 receiveMessage
>     src/lib/server.ts:103:58
> • server.ts:62 <anonymous>
>     src/lib/server.ts:62:53
> • node:events:527 emit
>     node:events:527:28
> • websocket.js:1160 receiverOnMessage
>     node_modules/ws/lib/websocket.js:1160:20
> • node:events:527 emit
>     node:events:527:28
> • receiver.js:528 dataMessage
>     node_modules/ws/lib/receiver.js:528:14

When looking at the changelog of eufy-sercurity-client, this should be fixed since version 1.1.0, but it doesn't seem to be working. Does anyone have an idea how this can be solved?

**Comment by fuatakgun** [comment]
Let's open an issue to track this, definitely a bug.
